packages/editor/src/client/panel/imageEditPanel.tsx

```diff
@@ -357,9 +363,6 @@ export function ImagedEditPanel({id, file, onClose, onSave}: ImageEditPanelProps
         <Button appearance={'primary'} disabled={isDisabled} onClick={() => handleSave()}>
           {isUpload ? t('images.panels.upload') : t('images.panels.save')}
         </Button>
-        <Button appearance={'subtle'} onClick={() => onClose?.()}>
```

Comment:
I would keep this button. Otherwise people only have the chance of the little x in the top right corner to cancel their changes. I recommend to change the appearance to red.